src/main/java/crazypants/enderio/machine/monitor/GuiPowerMonitor.java

```diff
@@ -309,6 +313,10 @@ private void renderInfoTab(int sx, int sy) {
       int sectionGap = SPACING;
 
       FontRenderer fontRenderer = getFontRenderer();
+      if (te.maxPowerInConduits == 0) {
+        fontRenderer.drawSplitString(noNetworkError, x, y, 170, ColorUtil.getRGB(Color.red));
+      } else {
+
       rgb = headingCol;
```

Comment:
I didn't want to indent the whole large block because github's diff is to stupid to display it as a simple whitespace change.

Also, I forgot.